ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -2657,6 +2657,505 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
is this supposed to be `always_run: true`?

Reply:
Marked all e2e tests as `always_run: false`. `okd-images` and `okd-unit`/`okd-verifycodegen` are flake-free and would run on every PR